src/rgw/rgw_rados.h

```diff
@@ -1541,4 +1558,220 @@ class RGWRados
 						     uint32_t num_shards);
 };
 
+
+struct get_obj_aio_data {
+  struct get_obj_data *op_data;
+  off_t ofs;
+  off_t len;
+};
+
+struct get_obj_io {
+  off_t len;
+  bufferlist bl;
+};
+
+class librados::CacheRequest {
+  public:
+    ceph::mutex lock;
```

Comment:
in the jenkins log: https://jenkins.ceph.com/job/ceph-perf-crimson/358/console and in local compilation with certain cmake flags
the following error occures:
![image](https://user-images.githubusercontent.com/31659604/88512959-0b7c0800-cff0-11ea-842b-d1ef96ab913f.png)

possibly mutex initialization as below or similar will mitigate it:
`ceph::mutex lock = ceph::make_mutex("CacheRequest");`